include/RED4ext/Dump/Reflection-inl.hpp

```diff
@@ -1082,11 +1081,22 @@ RED4EXT_INLINE void ClassFileDescriptor::EmitFile(std::filesystem::path aOutPath
 
         o << "namespace " << ns << std::endl;
         o << "{" << std::endl;
-        o << "struct " << name;
+    }
+
+    o << "struct ";
+
+    if (alignment == 0x10)
+    {
+        o << "__declspec(align(0x" << std::hex << std::uppercase << alignment << ")) ";
```

Comment:
I think this would start generating compiler warnings. Maybe it is a good idea to do the same we did for members, e.g. `pragma push`.

Reply:
Maybe `alignas`?

Reply:
It will be the same, we need to disable the compiler warning for that line. This is the warning: [C4324](https://learn.microsoft.com/en-us/cpp/error-messages/compiler-warnings/compiler-warning-level-4-c4324?view=msvc-170).

Reply:
I will merge this for now, and we can fix it later.